app/controllers/fhir/r4/api_controller.rb

```diff
@@ -1044,6 +1075,13 @@ def search_patients(options)
         query = query.where('primary_telephone like ?', Phonelib.parse(search, 'US').full_e164)
       when 'email'
         query = query.where('email like ?', "%#{search}%")
+      when 'birthdate'
+        query = query.where(date_of_birth: search)
+      when 'identifier'
+        (identifier, value) = search.split('|')
```

Comment:
```suggestion
        (identifier, value) = search.split('|', 2)
```
In theory I think a value could have `|` in it, so we want to just split on the first instance.

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1098/commits/dfabb5dca271e1d1f47b09680a1dd63cd63f6218#diff-e79e38176ee697a8541cffba23c60d9decaa69382ca5c6e875da033508e5b428R1091